### DISCUSSION TITLE
iOS 15 compatibility?

### DISCUSSION THREAD

**Discussion by ethank** [mKeRix/room-assistant#787] [Q&A]
I'm suspecting the answer is "not yet" but is the iOS companion iOS 15 compatible? I'm not seeing any errors and all permissions are on, but the phone isn't being seen by room assistant RPI.

**Comment by mKeRix** [comment]
I haven't tested it with iOS 15 yet - Bluetooth stuff needs to be tested on device, unfortunately I have no separate test device and don't want to install the beta on my personal phone. I looked around in the WWDC announcements and didn't see anything that would impact how room-assistant works, but that doesn't necessarily mean much.

If you'd like to dig deeper now the app is also open source on GitHub. In general I do want to support iOS 15 as well of course. :)
